### ISSUE TITLE
questions about VQA

### ISSUE THREAD

**Issue by Henry9805** [zdou0830/METER#7]
Hi, could you share the VQAv2 result fine-tuning with image resolution of 384, the result implemented by me is 76.52 and it is based on your checkpoint pretrained on COCO, SBU, VG, CC3M.

**Comment by zdou0830** [comment]
Hi, we didn't save this checkpoint. What are your training/testing commands and can you reproduce the result of our model with the image resolution of 576?

**Comment by Henry9805** [comment]
`python3 run.py with \
    data_root=vqa_arrows \
    num_gpus=8 \
    num_nodes=1 \
    task_finetune_vqa_clip_bert \
    per_gpu_batchsize=16 \
    load_path=meter_clip16_288_roberta_pretrain.ckpt \
    clip16 \
    text_roberta \
    image_size=384 \
    log_dir="finetune_vqa_384/" `

**Comment by Henry9805** [comment]
I will try to reproduce the vqa result of your model with the image resolution of 576 today,  I will tell you if when the result is reproduced.

**Comment by zdou0830** [comment]
Thanks! Could you try adding clip_randaug to the command?

**Comment by Henry9805** [comment]
I'm pretty sure clip_randaug is already adding to the training commands, the previous comment was wrong.

**Comment by zdou0830** [comment]
ok I'm testing the code on my end. just to confirm, are you testing the checkpoint `checkpoint_last.ckpt` in your result directory, and can you reproduce the score using the provided VQA-finetuned checkpoint?

**Comment by Henry9805** [comment]
I have reproduced the right vqa socre, which is 77.67, when using the VQA-fintuned checkpoint. And the testing checkpoint is the `checkpoint_last.ckpt`, since the best checkpoint is also the 'checkpoint_last.ckpt'.

**Comment by eladsegal** [comment]
Hi, I am also trying to reproduce the VQA results.
Can you please tell me which GPU did you use, and how long did the fine-tuning take? I get around ~15 hours per epoch with 8 RTX 3090 for the command in the README (image_size=576).

**Comment by eladsegal** [comment]
Hi, I am also trying to reproduce the results on VQA.
Can you please share which GPUs are you using, what are the values for `per_gpu_batchsize` and `image_size`, and how long does training take?

**Comment by Henry9805** [comment]
> Hi, I am also trying to reproduce the results on VQA. Can you please share which GPUs are you using, what are the values for `per_gpu_batchsize` and `image_size`, and how long does training take?

I am using A100 to reproduce the results. and the 'per_gpu_batchsize' is 4 when 'image_size' is 576, but if the 'image_size' is set to 384, 'per_gpu_batchsize' can be set to 16.

**Comment by zdou0830** [comment]
Hi @Henry9805, I just ran the command
`python run.py with data_root=$DATA_DIR num_gpus=8 num_nodes=1 task_finetune_vqa_clip_bert per_gpu_batchsize=8 clip16 text_roberta image_size=384 clip_randaug load_path=meter_clip16_288_roberta_pretrain.ckpt` and got a VQA score of 77.43 on test-dev. 

Right now the only difference I could see is the difference in `per_gpu_batchsize`. Could you set it to 8 and see if it would help?

**Comment by Henry9805** [comment]
@zdou0830 thanks, I notice that the 'val_check_interval' is also different I set it to 1.0, and could you please share the 'vqa' arrows?  i'm afraid that there is some difference between the data. it will be better if you can share 'snli' and 'nlvr' arrows.

**Comment by zdou0830** [comment]
'val_check_interval' shouldn't matter as I tested the last checkpoint. I'll see if I could share the data files.

**Comment by Henry9805** [comment]
@zdou0830 hi, I just noticed my VQAv2 label size is 3128, and it does not match the size you mention in your config file. I have tried to remake these arrows, but the size is still 3128.  is it right?

**Comment by zdou0830** [comment]
It is right and this is an issue in ViLT (https://github.com/dandelin/ViLT/issues/39).

**Comment by Henry9805** [comment]
thanks, I tried different settings. the results seem same as before. I want to check whether the data is broken in my end. please share some data arrows if you are convenient

**Comment by zdou0830** [comment]
I uploaded the VQA data files here: https://drive.google.com/file/d/1qT7YWHpLg-fAL43daKlOsYx2EbbQk--d/view?usp=sharing.

**Comment by Henry9805** [comment]
thank you very much, the data seems no different from mine. and the results is also same as before. I will try other settings.

**Comment by mactavish91** [comment]
> thank you very much, the data seems no different from mine. and the results is also same as before. I will try other settings.

@Henry9805 Hello, have you solved the problems you encountered before?

**Comment by Henry9805** [comment]
@mactavish91 I haven't solve this problem in the end and I have replaced my baseline model with another model

**Comment by mactavish91** [comment]
> @mactavish91 I haven't solve this problem in the end and I have replaced my baseline model with another model

@Henry9805 It seems that you set the resolution to 384 when fine-tuning and achieved an accuracy rate of 76.52. Did you improve the experimental results after raising the resolution to 576? It will definitely be improved according to common sense.
